src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3163,9 +3163,8 @@ private static bool IsValidExpressionBody(SyntaxNode expressionSyntax, BoundExpr
             Binder bodyBinder = this.GetBinder(expressionBody);
             Debug.Assert(bodyBinder != null);
 
-            RefKind refKind = RefKind.None;
-            ExpressionSyntax expressionSyntax = expressionBody.Expression.CheckAndUnwrapRefExpression(diagnostics, out refKind);
-            BindValueKind requiredValueKind = GetRequiredReturnValueKind(refKind);
+            ExpressionSyntax expressionSyntax = expressionBody.Expression.CheckAndUnwrapRefExpression(diagnostics, out var refKind);
```

Comment:
>var [](start = 119, length = 3)

Please revert the change. At the very least the type of the out declaration should be spelled out